admin/pre_reg/serializers.py

```diff
@@ -53,4 +51,36 @@ def serialize_draft_registration(draft, json_safe=True):
         'title': draft.registration_metadata['q1']['value'],
         'embargo': embargo,
         'registered_node': node_url,
+        'status': status,
+        'logs': draft.logs,
     }
+
+
+def get_url(draft):
+    url = furl(OSF_DOMAIN)
+    if draft.registered_node is not None:
+        url.path.add(draft.registered_node.url)
+        return url.url
+    elif draft.branched_from is not None:
+        url.path.add(draft.branched_from.url)
+        return url.url
+    return None
+
+
+def get_embargo(draft, json_safe):
+    registration_choice = draft.approval.meta.get('registration_choice', None)
+    if registration_choice == EMBARGO:
+        time = parser.parse(draft.approval.meta['embargo_end_date'])
+        return iso8601format(time) if json_safe else time
```

Comment:
FYI, [this](https://github.com/django/django/blob/master/django/core/serializers/json.py#L89) automatically supports encoding datetimes into JSON. 

Usage: `json.dumps(temp_output, cls=DjangoJSONEncoder)`